### DISCUSSION TITLE
Lithium plating submodules

### DISCUSSION THREAD

**Discussion by abernabeu** [pybamm-team/PyBaMM#2239] [General]
Hi PyBaMM comunity,
I'm trying to replicate the code developed at the course _Plug & Play Physics on PybaMM_, in particular the _Lithium plating submodules_ and I'm getting the following error:

**AttributeError: module 'pybamm.parameters.parameter_sets' has no attribute 'Chen2020_plating'**

The code used:

import pybamm

model1 = pybamm.lithium_ion.DFN(options={"lithium plating": "reversible"})
model2 = pybamm.lithium_ion.DFN(options={"lithium plating": "irreversible"})

chemistry = pybamm.parameter_sets.Chen2020_plating
parameter_values = pybamm.ParameterValues(chemistry=chemistry)
parameter_values.update({"Ambient temperature [K]":268.15})
parameter_values.update({"Upper voltage cut-off [V]":4.21})
parameter_values.update({"Lithium plating kinetic rate constant [m.s-1]":1E-9})

Could you give me hand?
Yours faithfully,
Andrés

**Comment by brosaplanella** [comment]
Hi Andrés, in the most recent release Chen2020_plating has been renamed OKane2022 to match the paper with the details on the parameters.

**Comment by abernabeu** [comment]
Dear Ferran,
Thank you very much for the help, have a nice day!
Yours faithfully,
Andrés
